hazelcast/db.py

```diff
@@ -175,6 +230,12 @@ def executemany(self, operation: str, seq_of_params: Sequence[Tuple]) -> None:
             _wrap_error(fut.result)
 
     def fetchone(self) -> Optional[SqlRow]:
+        """Fetches a single row from the result
+
+        Returns:
+            A single row if there are rows in the result or None.
+
```

Comment:
```suggestion
```

Reply:
No, the empty newline after the return block

Reply:
https://github.com/hazelcast/hazelcast-python-client/pull/626/commits/c8680b8b8235c89cfff4276c07a5d5840b1bf404